<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Locally</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/img/icon.png</normaloff>:/img/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="res.qrc">:/img/up.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Uploads</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="resumeSenderBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Resume</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/resume.png</normaloff>:/img/resume.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pauseSenderBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Pause</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/pause.png</normaloff>:/img/pause.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancelSenderBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Cancel selected item</string>
            </property>
            <property name="text">
             <string>Cancel</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/cancel.png</normaloff>:/img/cancel.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="toolTip">
             <string>Clear completed items</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/clear.png</normaloff>:/img/clear.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="senderTableView">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="res.qrc">:/img/down.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Downloads</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="resumeReceiverBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Resume</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/resume.png</normaloff>:/img/resume.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pauseReceiverBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Pause</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/pause.png</normaloff>:/img/pause.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancelReceiverBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Cancel selected item</string>
            </property>
            <property name="text">
             <string>Cancel</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/cancel.png</normaloff>:/img/cancel.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="toolTip">
             <string>Clear completed items</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/clear.png</normaloff>:/img/clear.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="receiverTableView">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>58</width>
     <height>38</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSend"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
  </widget>
  <action name="actionSend">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/send.png</normaloff>:/img/send.png</iconset>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select and send files&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/settings.png</normaloff>:/img/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSend</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSendActionTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSettingsActionTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>senderTableView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSenderTableDoubleClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>receiverTableView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>onReceiverTableDoubleClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSenderClearClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>908</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>709</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onReceiverClearClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>908</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelSenderBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSenderCancelClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>815</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseSenderBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSenderPauseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>772</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>1043</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resumeSenderBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSenderResumeClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>733</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>932</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelReceiverBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onReceiverCancelClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>818</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>952</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseReceiverBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onReceiverPauseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>777</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>930</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resumeReceiverBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onReceiverResumeClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>734</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>925</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>senderTableView</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSenderTableContextMenuRequested(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>-27</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>receiverTableView</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>MainWindow</receiver>
   <slot>onReceiverTableContextMenuRequested(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>-35</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSendActionTriggered()</slot>
  <slot>onSettingsActionTriggered()</slot>
  <slot>onAboutActionTriggered()</slot>
  <slot>onSenderTableDoubleClicked(QModelIndex)</slot>
  <slot>onReceiverTableDoubleClicked(QModelIndex)</slot>
  <slot>onSenderClearClicked()</slot>
  <slot>onReceiverClearClicked()</slot>
  <slot>onSenderCancelClicked()</slot>
  <slot>onSenderPauseClicked()</slot>
  <slot>onSenderResumeClicked()</slot>
  <slot>onReceiverCancelClicked()</slot>
  <slot>onReceiverPauseClicked()</slot>
  <slot>onReceiverResumeClicked()</slot>
  <slot>onSenderTableContextMenuRequested(QPoint)</slot>
  <slot>onReceiverTableContextMenuRequested(QPoint)</slot>
 </slots>
</ui>
